.vscode/launch.json

```diff
@@ -5,17 +5,19 @@
 			"type": "chrome",
 			"request": "launch",
 			"name": "Playground: Browser",
-			"url": "http://localhost:10001"
+			"url": "http://localhost:10001",
+			"webRoot": "${workspaceFolder}"
 		},
 		{
 			"type": "node",
 			"request": "launch",
 			"runtimeArgs": ["--watch"],
 			"name": "Playground: Server",
 			"outputCapture": "std",
-			"program": "start.js",
+			"program": "${workspaceFolder}/packages/playground/start.js",
 			"cwd": "${workspaceFolder}/packages/playground",
-			"cascadeTerminateToConfigurations": ["Playground: Browser"]
+			"restart": true,
+			"console": "integratedTerminal"
```

Comment:
This will make the output appear in the terminal instead of the debug console, is there any benefit to doing so?